Android/app/src/main/java/app/intra/net/go/GoProber.java

```diff
@@ -38,22 +37,13 @@ public GoProber(Context context) {
   @Override
   public void probe(String url, Callback callback) {
     new Thread(() -> {
-      String dohIPs = GoVpnAdapter.getIpString(context, url);
+      String dohIPs = GoVpnAdapter.getDoHServerFallbackIPsString(context, url);
       try {
         // Protection isn't needed for Lollipop+, or if the VPN is not active.
-        Protector protector = VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP ? null :
+        SocketProtector protector = VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP ? null :
```

Comment:
Let's get rid of the protector, it complicates things and is no longer needed.

I took a look in the Play Store, and only 0.2% of the installed audience is below Lollipop (Android 5): https://docs.google.com/spreadsheets/d/15zsfgMhJ7TsLumE6S5ZykETwMBvrBptFllX0Z02gFPw/edit?usp=sharing&resourcekey=0-inA8jaG0XS58zXNXU9YOzQ

The global stats say 0.66%: https://apilevels.com/

It's not worth the trouble.

Let's make that change separately though. Change, then move.

Reply:
As we remove the protector, we should also update the minSdkVersion to 21 (Lollipop), to make sure older phones don't get the new version that will break.